cert/selfsigned.go

```diff
@@ -210,21 +211,22 @@ func GenCertPair(org, certFile, keyFile string, tlsExtraIPs,
 	// Generate a serial number that's below the serialNumberLimit.
 	serialNumber, err := rand.Int(rand.Reader, serialNumberLimit)
 	if err != nil {
-		return fmt.Errorf("failed to generate serial number: %s", err)
+		return nil, nil, fmt.Errorf("failed to generate serial number:"+
```

Comment:
micro nit: if we have to break an error string between words, we usually try to keep the space on the previous line. Since this would make the line too long here, I personally would break before the word "number".